### PR TITLE
Add Linux preset for Visual Studio

### PR DESCRIPTION
**Pull request by twollgam**
### What does this PR do?

Add Linux preset for Visual Studio to allows to work with wsl2 (for example ubuntu) in Visual Studio

### Closes Issue(s)

None

### Motivation

Enable more options for developers

### Additional Notes

Will need not testing cause no game code is involved.

### CODE REVIEW

File: CMakePresets.json [hunk 1]
```diff
@@ -45,8 +45,28 @@
...
+      "name": "linux-default",
+      "displayName": "Linux Debug",
+      "description": "Sets Ninja generator, compilers, build and install directory, debug build type",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/out/build/${presetName}",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "SELF_BUILT_AUBIO": "ALWAYS",
+        "SELF_BUILT_CED": "ALWAYS",
+        "CMAKE_INSTALL_PREFIX": "${sourceDir}/out/install/${presetName}"
```
Comment (Baklap4) [inline CMakePresets.json:59]: Please use the same structure as windows
```suggestion
       "CMAKE_INSTALL_PREFIX": "${sourceDir}/build/${presetName}-install"
```

File: CMakePresets.json [hunk 2]
```diff
@@ -45,8 +45,28 @@
...
       }
-    }
-  ],
+    },
+    {
+      "name": "linux-default",
+      "displayName": "Linux Debug",
+      "description": "Sets Ninja generator, compilers, build and install directory, debug build type",
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/out/build/${presetName}",
```
Comment (Baklap4) [inline CMakePresets.json:54]: Please use the same structure as windows
```suggestion
      "binaryDir": "${sourceDir}/build/${presetName}",
```

File: CMakePresets.json [hunk 3]
```diff
@@ -45,8 +45,28 @@
...
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
```
Comment (Baklap4) [inline CMakePresets.json:56]: please create the same structure as windows:

a "base" for linux and then per cmake_build_type a target :)

This makes for easy switching between release/debug whilst not having to duplicate the code.

**Review by Baklap4 — CHANGES_REQUESTED** [review]
Please also add the buildPresets in a similar manner

File: CMakePresets.json (hunk 1 above)
Comment (twollgam) [reply CMakePresets.json:59]: 👍

**Review by Baklap4 — APPROVED** [review]

**Review by Baklap4 — APPROVED** [review]

**Review by ooshlablu — APPROVED** [review]

**Pull request merged**